docs/topics/mta-review-reports.adoc

```diff
@@ -71,6 +71,7 @@ The dashboard gives an overview of the entire application migration effort. It s
 
 .Dashboard
 image::3-3-dashboard.png[Dashboard]
+// The image I have from the static HTML file does not show any Issues or Share archives. Is there a sample application that I can run that shows these?

```

Comment:
The image from my static HTML file does not show any issues in the Dashboard.
Is there a sample application that an analysis will produce issues, shared archives, etc?